ldi-orchestrator/ldio-connectors/ldio-ngsiv2-to-ld-adapter/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioNgsiv2ToLdAdapterAutoConfig.java

```diff
@@ -11,7 +11,7 @@
 public class LdioNgsiv2ToLdAdapterAutoConfig {
 
 	@SuppressWarnings("java:S6830")
-	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldi.NgsiV2ToLdAdapter")
+	@Bean("Ldio:NgsiV2ToLdAdapter")
 	public LdioAdapterConfigurator ldiHttpOutConfigurator() {
```

Comment:
This is more outside of the scope of this story, but I would expect this method name to be more like `ldioAdapterConfigurator`